index.html

```diff
@@ -3881,10 +3881,21 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="image">
+			<rdef>image</rdef>
+			<div class="role-description">
+				<p>A container for a collection of <a>elements</a> that form an image. See synonym <rref>img</rref>.</p>
+				<div class="note" id="role-none-note-none">
```

Comment:
Probably want to change that id because it already exists for the note in role none. :)

Reply:
yeh.... i suppose that wouldn't be such a bad idea....